### PR TITLE
*: fix inconsistent spelling "Sql"

### PR DESCRIPTION
**Pull request by dragonly**
Signed-off-by: dragonly <liyilongko@gmail.com>

### What problem does this PR solve?

fix inconsistent spelling "Sql"

Problem Summary:

### What is changed and how it works?

```bash
$ grep Sql . -RnI
./ddl/db_integration_test.go:2221:func (s *testIntegrationSuite3) TestSqlFunctionsInGeneratedColumns(c *C) {
./executor/seqtest/prepared_test.go:249:		_, err = tk.Exec("prepare stmt FROM @Sql")
```

What's Changed:
This PR just fix those two inconsistent spelling "Sql"

How it Works:

### Related changes

None

### Check List 

Tests 

- Unit test
- Integration test

Side effects

None

### Release note 

- `No release note`

### CODE REVIEW

File: executor/seqtest/prepared_test.go [hunk 1]
```diff
@@ -246,7 +246,7 @@ func (s *seqTestSuite) TestPrepared(c *C) {
 		_, err = tk.Exec("execute stmt")
 		c.Assert(err, IsNil)
 		tk.MustQuery("select a from prepare1;").Check(testkit.Rows("6"))
-		_, err = tk.Exec("prepare stmt FROM @Sql")
+		_, err = tk.Exec("prepare stmt FROM @SQL")
```
Comment (wjhuang2016) [inline executor/seqtest/prepared_test.go:249]: I guess `Sql` is by design here.

Comment (dragonly) [reply executor/seqtest/prepared_test.go:249]: Sorry I didn't get it, could you please elaborate a little bit, Thanks!

Comment (wjhuang2016) [reply executor/seqtest/prepared_test.go:249]: This case wants to test an upper case char in `sql`, so it uses `Sql`.

Comment (wjhuang2016) [reply executor/seqtest/prepared_test.go:249]: FYI: https://github.com/pingcap/tidb/pull/19373

Comment (wjhuang2016) [reply executor/seqtest/prepared_test.go:249]: If you change it to `SQL`, then it will duplicate with L244.

Comment (dragonly) [reply executor/seqtest/prepared_test.go:249]: > This case wants to test an upper case char in `sql`, so it uses `Sql`.
Oh I see. Then I suggest not using `Sql` here, which is misleading, such as in this thread :D
What about changing it to this
```suggestion
		_, err = tk.Exec("prepare stmt FROM @Uppercase")
```

Comment (wjhuang2016) [reply executor/seqtest/prepared_test.go:249]: ok

**Comment by xhebox** [comment]
@dragonly any update?

**Comment by dragonly** [comment]
@xhebox Sorry to reply late. It's done, PTAL

File: executor/seqtest/prepared_test.go [hunk 2]
```diff
@@ -248,7 +248,7 @@ func (s *seqTestSuite) TestPrepared(c *C) {
 		_, err = tk.Exec("execute stmt")
 		c.Assert(err, IsNil)
 		tk.MustQuery("select a from prepare1;").Check(testkit.Rows("6"))
-		_, err = tk.Exec("prepare stmt FROM @Sql")
+		_, err = tk.Exec("prepare stmt FROM @Uppercase")
```
Comment (xhebox) [inline executor/seqtest/prepared_test.go:251]: It breaks the test, there is no `uppercase` user variable.

**Comment by tisonkun** [comment]
I think `Sql` and `SQL` are both used among many projects. I don't have a strong feeling on the spell and we can hardly guard any following contributor spell "correct".

Pending to close this. What do you think @dragonly ?

**Comment by dragonly** [comment]
@tisonkun LGTM, will close this PR.

**Pull request closed without merging**